package.json

```diff
@@ -59,14 +56,13 @@
   ],
   "scripts": {
     "example:basic": "cd boilerplates/react-union-boilerplate-basic && yarn start",
-    "lint": "eslint \"packages/*.js\" \"packages/**/*.js\" \"boilerplates/**/*.js\"",
+    "lint": "eslint \"packages/*.js\" \"packages/**/*.js\" \"boilerplates/**/*.js\"^7.0.0",
```

Comment:
Is this a mistake? It looks really weird.

Reply:
yes, it is